Assets/Scripts/BulletManagement/Bullet.cs

```diff
@@ -1,53 +1,56 @@
 ﻿using UnityEngine;
 
-/**
- * Base class for bullets. Bullets are destroyed when they no longer are
- * in the view of the Camera.
- */
-public class Bullet : MonoBehaviour
+namespace BulletManagement
 {
-    public float decayTime = 1f;
-    public float instantiationTime = 0.0f;
+    /**
+    * Base class for bullets. Bullets are destroyed when they no longer are
+    * in the view of the Camera.
+    */
+    public class Bullet : MonoBehaviour
+    {
+        public float decayTime = 1f;
+        public float instantiationTime;
 
-    // Metadata
-    public Emitter emitter = null;
-    public BulletPattern bulletPattern = null;
-    private int bulletId;
+        // Metadata
+        public Emitter emitter;
+        public BulletPattern bulletPattern;
+        private int bulletId;
 
-    void Start()
-    {
-        // Destroy self in N seconds after emitt
-        // Do not do so if set to zero.
-        if (decayTime != 0)
+        private void Start()
         {
-            Destroy(this.gameObject, decayTime);
+            // Destroy self in N seconds after emitt
+            // Do not do so if set to zero.
+            if (decayTime != 0)
+            {
+                Destroy(this.gameObject, decayTime);
+            }
+            instantiationTime = Time.timeSinceLevelLoad;
         }
-        this.instantiationTime = Time.timeSinceLevelLoad;
-    }
 
-    void OnBecameInvisible() {
-        Destroy(this.gameObject);
-    }
+        private void OnBecameInvisible() {
+            Destroy(gameObject);
+        }
 
-    void Update()
-    {
-        float timeAlive = (Time.timeSinceLevelLoad - this.instantiationTime);
+        void Update()
+        {
+            var timeAlive = (Time.timeSinceLevelLoad - instantiationTime);
```

Comment:
What's your reasoning behind using `var`?

Reply:
The type can be inferred, so there's no need to specify it. Standard C# convention.

Reply:
This is OK imo because we are dealing with `Time` which is abstracted.